docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -130,22 +135,23 @@ The *Roles* realm resource type represents the roles in a realm. You can manage
 
 | *map-role*               | Defines if a realm administrator can assign a role (or multiple roles) to a user.
 | *map-role-composite*     | Defines if a realm administrator can assign a role (or multiple roles) as a composite to another role.
-| *map-role-client-scope*  | Defines if a realm administrator can apply a role (or multiple roles) to an another client’s scope.
+| *map-role-client-scope*  | Defines if a realm administrator can apply a role (or multiple roles) to a client scope.
 |===
 
+The *map-roles* operation does not grant the ability to manage users or assign roles arbitrarily. The administrator must also
+have user role mapping permissions on the user.
+
 [NOTE]
 ====
-* The *map-role* operation does not grant the ability to manage users or assign roles arbitrarily; the administrator must also have
-  user role mapping permissions.
-* When mapping client roles, if an administrator has permission to *map-roles*, *map-roles-composite*, or *map-roles-client-scope* for
+* TODO: When mapping client roles, if an administrator has permission to *map-roles*, *map-roles-composite*, or *map-roles-client-scope* for
```

Comment:
Is TODO a placeholder?

Reply:
It is not, I would like to discuss this text with @vramik to understand it better. I'll update this one as soon as I have an answer from him.

Reply:
I believe it's about the fact that if you have client permission for `map-roles` , `map-roles-composite` or `map-roles-client-scope`  you won't reach point where permissions for individual roles are evaluated.

Reply:
Updated the text.